code/WorkInProgress/TempEngine.dm

```diff
@@ -13,10 +31,20 @@
 
 	var/side = null // 1=left 2=right
 	var/last_pressure_delta = 0
-
+	var/static/list/circulator_accepted_reagents = list("oil"=1.0,"lube"=1.1,"super_lube"=1.12)
```

Comment:
it would be cool if we could clog the engine with stuff ex:
carbon, vomit, etc.
also can we lubricate the engine with blood please ty

Reply:
could also base it off reagent viscosity?

Reply:
I misinterpreted the property comment in that it was used for other interactions plus lack of it being set across reagents and using the default made me shy away.

Using viscosity would solve the bug I mentioned above and make calculation more consistent for items not in the list.  And then a follow on issue to examine reagent viscosity?

Reply:
Added support for viscosity scaling differently based on reagent state